src/PleskX/Api/Operator/SiteAlias.php

```diff
@@ -32,4 +32,20 @@ public function create(array $properties, array $preferences = [])
         return new Struct\Info($response);
     }
 
+	/**
+	 * @param string $siteName
+	 * @return Struct\Info
+	 */
+	public function delete(string $siteName)
```

Comment:
Since we support PHP 5.6 in Plesk version before 17.8, we cannot use type-hinting for the primitive types like `string`.